shoes-core/lib/shoes/line.rb

```diff
@@ -19,6 +19,16 @@ def create_dimensions(point_a, point_b)
       style[:y2] = point_b.y
     end
 
+    def in_bounds?(x, y)
+      left_p, right_p = [point_a, point_b].sort_by { |a, b| [a.x, a.y] }
+      dx = x - left_p.x
+      dy = y - left_p.y
+      dyb = right_p.y - left_p.y
+      dxb = right_p.x - left_p.x
```

Comment:
Too bad I'm very anxious about the '80 chars limit' :) How about ydistance_right_left_points/xdistance_right_left_points or hight_right_left_points/width_right_left_points?